starter_scripts/exaslct_within_docker_container.sh

```diff
@@ -22,6 +22,7 @@ fi
 
 SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 declare -a mount_point_paths
+# shellcheck disable=SC2207
```

Comment:
should we add a comment, why we disable here?

Reply:
Done